### PR TITLE
feat: upgrade modal

### PR DESCRIPTION
**Pull request by Leiksa**
👀 [Visit Preview](https://v5-modal--welcome-ui.netlify.app)

### CODE REVIEW

File: docs/pages/components/modal.mdx [hunk 1]
```diff
@@ -93,21 +96,28 @@ function() {
...
   const modal = useModalState()
+  const title = "Nullam non lacinia"
+  const subtitle = "Praesent sit amet quam ac velit faucibus dapibus, quisque sapien ligula."
+  const icon = <ActionsIcon size="xl" color="dark.900" />
+
+  const informations = {
+    title: "Donec id tellus",
+    subtitle: "Sed lorem lacus, hendrerit non sodales id, consectetur quis magna. Mauris auctor blandit porta. In imperdiet rutrum nunc."
+  }
+
```
Comment (theo-mesnil) [inline docs/pages/components/modal.mdx:112]: Can you please add more examples with differents header and footer ? or create a selector to show/not show informations ?

File: packages/Core/theme/modals.ts [hunk 2]
```diff
@@ -54,6 +74,5 @@ export const getModals = (theme: WuiTheme): ThemeModals => {
       },
       auto: {},
     },
-    cover: {},
```
Comment (theo-mesnil) [inline packages/Core/theme/modals.ts:57]: We need to keep cover on our DS even if we don't use it on our projects :)

File: packages/Modal/Close.tsx [hunk 3]
```diff
@@ -5,8 +5,8 @@ export const Close: React.FC<CloseButtonProps> = props => {
   return (
     <CloseButton
       position={{ xs: 'fixed', md: 'absolute' }}
-      right={20}
-      top={20}
+      right={8}
```
Comment (theo-mesnil) [inline packages/Modal/Close.tsx:8]: maybe can we get this from the spaces theme ?

File: packages/Modal/index.tsx [hunk 4]
```diff
@@ -115,65 +129,16 @@ const ModalComponent = forwardRef<'div', ModalProps>(
...
 
-const Cover: React.FC<ShapeProps> = props => {
-  const { modals } = useTheme()
-
-  return (
-    <div>
-      <Shape {...modals.cover} {...props} />
-    </div>
-  )
-}
```
Comment (theo-mesnil) [inline packages/Modal/index.tsx:153]: Keep this on the component

**Review by theo-mesnil — COMMENTED** [review]
😻

File: docs/pages/upgrade.mdx [hunk 5]
```diff
@@ -6,6 +6,12 @@ Version 5: Rework of our foundations
 
 ### Theme
 
+#### Modal
+
+- We have adjust colors and spacing.
+- **Modal.Title** is now named **Modal.Header** and have props `title` (mandatory) and `subtitle` (optional).
+- **Modal.Cover** has been removed.
+
```
Comment (cnairiwttj) [inline docs/pages/upgrade.mdx:14]: Sorry, my comment is not so useful but the correct sentences are:
- We have adjust**ed**...
- ...and ha**s** props `title`

**Review by cnairiwttj — APPROVED** [review]

**Review by theo-mesnil — APPROVED** [review]

**Pull request merged**